packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+	<div class="container">
+		<!-- <i class="pi pi-magic" /> -->
+		<InputText
+			class="input"
+			ref="inputElement"
+			v-model="queryString"
+			type="text"
+			:disabled="kernelStatus === KernelState.busy"
+			:placeholder="kernelStatus ? 'Please wait...' : 'What do you want to do?'"
+			@keydown.enter="submitQuery"
+		/>
+		<Dropdown :disabled="true" :model-value="contextLanguage" :options="contextLanguageOptions" />
+		<i v-if="kernelStatus === KernelState.busy" class="pi pi-spin pi-spinner kernel-status" />
+		<Button v-else icon="pi pi-send" />
+	</div>
+</template>
+
+<script setup lang="ts">
+import InputText from 'primevue/inputtext';
+import Button from 'primevue/button';
+import { ref, onUnmounted, onMounted } from 'vue';
+import { KernelState, KernelSessionManager } from '@/services/jupyter';
+import Dropdown from 'primevue/dropdown';
+
+const props = defineProps<{
+	context: string;
+	contextInfo: any;
+}>();
+
+const emit = defineEmits(['append-output']);
```

Comment:
Might be better to rename `llm-output` or something like that, we already have `append-output-port` elsewhere so just want to have clear separation. It is also not clear what `append` is referring to because you can very well use the LLM-input-box for something completely different than appending code to a textbox.

Reply:
Good suggestion, the name was a relic of earlier versions